### PR TITLE
Text from Steve/Austin

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

File: data/siteText.yaml [hunk 1]
```diff
@@ -62,7 +61,7 @@ envEffects-p2: >-
   nonylphenol, and antimicrobials have been associated with cancer and endocrine disruption. 
   The high sorption capacity of plastic enables the accumulation of persistent organic pollutants 
   such as polychlorinated biphenyls, polycyclic aromatic hydrocarbons, and organochlorine pesticides, 
-  with concentrations of 10^5 – 10^6 times higher than in the surrounding water column. Trace metals 
+  with concentrations of 10<sup>5</sup> – 10<sup>6</sup> times higher than in the surrounding water column. Trace metals
```
Comment (ldecicco-USGS) [inline data/siteText.yaml:64]: @jiwalker-usgs this didn't seem to work for me locally

Comment (jiwalker-usgs) [inline data/siteText.yaml:64]: ah, it is probably escaping the html bits, we should be doing `{{{ }}}` for all the text, but this one might be missing it.

Comment (jiwalker-usgs) [inline data/siteText.yaml:64]: hmm, looking at it actually it is fine, I'll check it out

Comment (jiwalker-usgs) [inline data/siteText.yaml:64]: It works for me, are you being bit by the layout.yaml remake cache issue?

**Pull request merged**